lib/http.js

```diff
@@ -187,7 +187,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     validate('bedrock-profile-http.delegateCapability'),
     asyncHandler(async (req, res) => {
       const {account: {id: accountId}} = (req.user || {});
-      const {invoker, account} = req.body;
+      const {invoker, account, expires} = req.body;
```

Comment:
I don't think we want to allow `expires` to be set by the client. It's an attack vector, let's remove that and always set it to the configured value.